### PR TITLE
serializing and de-serializing models

### PR DESCRIPTION
**Pull request by Param-29**
## Description

This PR is a POC for a floating idea. This demonstrates how we could send trained models to the virtual machine and get it back. 

## Affected Dependencies

None

## How has this been tested?
1. Added in unit tests 
2. Local machine and CI 

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: src/syft/lib/sklearn/serializing_models.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        proto: Serialized version of model, which will be used to re-construct model.
+
+    Returns:
+        Re-constrcuted model.
+    """
+    vars_dict = Dict._proto2object(proto=proto.model)
+    ret_model = sklearn.linear_model.LogisticRegression()
+
+    for attribute, value in vars_dict.items():
+        if str(type(value)).find("_SyNone") != -1:
```
Comment (madhavajay) [inline src/syft/lib/sklearn/serializing_models.py:45]: Shouldn't we just upcast all the variables so that the output has no Syft types inside it.

```
if hasattr(value, "upcast"):
    value = value.upcast()
```

Or something?

**Review by madhavajay — APPROVED** [review]
Great thinking. Let's discuss how we can approach this more programmatically for future lib support changes.

**Comment by Param-29** [comment]
> Great thinking. Let's discuss how we can approach this more programmatically for future lib support changes.

Yeah, what I was thing was some kind of `SklearnModelWrapper` for modular code, lets discuss it sometime soon. Thanks for the help! :)
